lib/providers/order/uniswapxService.ts

```diff
@@ -14,23 +16,41 @@ export class UniswapXServiceProvider implements OrderServiceProvider {
     this.log = _log.child({ quoter: 'UniswapXOrderService' });
   }
 
-  async postOrder(order: Order, signature: string, quoteId?: string): Promise<void> {
+  async postOrder(order: Order, signature: string, quoteId?: string): Promise<ErrorResponse | void> {
```

Comment:
It feels kinda strange to have a method that only returns errors and nothing on success. Does it make sense to have it return an object too in the case of success and then we can check for error in the caller?